conda/activate.py

```diff
@@ -84,9 +106,18 @@ def __init__(self, shell, arguments=None):
             self.command_join = '\n'
 
             self.unset_var_tmpl = 'del $%s'
+            self.unexport_var_tmpl = self.unset_var_tmpl
             self.export_var_tmpl = "$%s = '%s'"
+            self.set_var_tmpl = self.export_var_tmpl
             self.run_script_tmpl = 'source "%s"'
 
+            self.post_tmpl = dedent("""\
```

Comment:
I'm not big fan of the line-continuation characters in Python code.
I'd rather just
```python
self.post_tmpl = dedent("""
...
""".strip("\n"))
```
or
```python
self.post_tmpl = "\n".join((
    "line1",
    "line2",
))
```

Reply:
I totally agree, I still haven't quite decided what I prefer. Will reconsider this.